server/routerlicious/packages/services-core/src/mongoCheckpointRepository.ts

```diff
@@ -62,6 +62,21 @@ export class MongoCheckpointRepository implements ICheckpointRepository {
 		documentId: string,
 		tenantId: string,
 	): { _id: string; documentId: string } & any {
+		const isError = !documentId || !tenantId;
+
+		if (isError) {
+			const error = new Error(`Cannot create filter due to missing parameter`);
+			Lumberjack.error(
+				"Missing parameter when writing checkpoint.",
+				{
+					...getLumberBaseProperties(documentId, tenantId),
+					documentIdInput: documentId,
+					tenantIdInput: tenantId,
+				},
```

Comment:
Do we need `documentIdInput` and `tenantIdInput` here?

Reply:
No, I removed them